servicetalk-grpc-protoc/build.gradle

```diff
@@ -115,10 +115,9 @@ task testJavadoc(type: Javadoc) {
   source = protobuf.generatedFilesBaseDir
   classpath = sourceSets.test.compileClasspath
   destinationDir = file("$buildDir/tmp/testjavadoc")
-  options.addBooleanOption("Xdoclint:syntax", true)
-  options.addBooleanOption("Xdoclint:html", true)
-  options.addBooleanOption("Xdoclint:accessibility", true)
-  // Don't use Xdoclint:missing for now because not all methods/types have javadocs.
+  options.addBooleanOption("Xwerror", true)
```

Comment:
This option will break our build because of how protobuf generate its javadoc. Let's defer adding these for now.

Reply:
From build logs: https://github.com/apple/servicetalk/runs/5491554228?check_suite_focus=true
```
> Task :servicetalk-grpc-netty:testClasses
[678](https://github.com/apple/servicetalk/runs/5491554228?check_suite_focus=true#step:7:678)
/home/runner/work/servicetalk/servicetalk/servicetalk-grpc-protoc/build/generated/sources/proto/test/java/test/shared/TestShared.java:1505: warning: @return has already been specified
[679](https://github.com/apple/servicetalk/runs/5491554228?check_suite_focus=true#step:7:679)

[680](https://github.com/apple/servicetalk/runs/5491554228?check_suite_focus=true#step:7:680)
       * @return This builder for chaining.
[681](https://github.com/apple/servicetalk/runs/5491554228?check_suite_focus=true#step:7:681)
         ^
[682](https://github.com/apple/servicetalk/runs/5491554228?check_suite_focus=true#step:7:682)
> Task :servicetalk-grpc-protoc:testJavadoc
[683](https://github.com/apple/servicetalk/runs/5491554228?check_suite_focus=true#step:7:683)
error: warnings found and -Werror specified
[684](https://github.com/apple/servicetalk/runs/5491554228?check_suite_focus=true#step:7:684)
1 error
[685](https://github.com/apple/servicetalk/runs/5491554228?check_suite_focus=true#step:7:685)
1 warning
```

Reply:
https://github.com/protocolbuffers/protobuf/issues/9612